src/services/SyncServiceProvider.js

```diff
@@ -42,7 +42,7 @@ export default function createSyncServiceProvider({ ydoc, syncService, fileId, i
 		'ws://localhost:1234',
 		'file:' + fileId,
 		ydoc,
-		{ WebSocketPolyfill },
+		{ WebSocketPolyfill, disableBc: true },
```

Comment:
Nice, maybe something to still keep in debug mode?

Reply:
I wonder how we could best trigger this from the dev environment.

Reply:
At another place we use `window._oc_debug` for this: https://github.com/nextcloud/text/blob/519637de1ccd75f9620060ff25b8f8f758da8a61/src/editor.js#L98

Reply:
I pushed a commit to set `disableBc` according to `window?._oc_debug`.

Reply:
Interesting, this broke our `recovers from a dropped message` Cypress test.